ods_ci/tests/Resources/OCP.resource

```diff
@@ -250,3 +250,11 @@ Delete Namespace From Openshift
         ${rc}   ${output}=    Run And Return Rc And Output    oc delete project ${namespace}
         Should Be Equal    "${rc}"    "0"   msg=${output}
     END
+
+Check If Pod Does Not Exist
+    [Documentation]    Check if pod does not exist using pod label-selector
+    [Arguments]        ${label_selector}    ${namespace}
+
+    ${rc}   ${output}=    Run And Return Rc And Output
+    ...    oc get pod -l {label_selector} -n ${namespace}
+    Should Be Equal    "${rc}"    "1"    msg=${output}
```

Comment:
cannot you use the kw `Is Resource Present`?

Reply:
Using Is Resource Present to check if the pod exists would need first to get the pod name (using label selector), and then checking if resource is present. This approach is a bit more efficient.